common/components/identity-bar/template.njk

```diff
@@ -16,6 +16,26 @@
           <p class="app-identity-bar__summary">
             {{ params.summary.html | safe if params.summary.html else params.summary.text }}
           </p>
+
+          {% if params.isLockedOut %}
+            {% for journey in params.journeys %}
+              <p class="app-identity-bar__summary">
+                and from <b>{{ journey.from_location.title }}</b>
+                to
+                {% if journey.to_location.title %}
+                  <b>{{ journey.to_location.title }}</b>
+                {% else %}
+                  <b> (awaiting destination) </b>
+                {% endif %}
```

Comment:
A journey will always have a `to_location` and a `from_location`, what we might see instead in the case of a lockout is that the journeys itself may not exist yet. The supplier might be aware of the lockout, but won't have scheduled the journeys yet.

So I think the condition here should be, if it's a lockout, but the move has still only got 0 or 1 journeys, to show `(awaiting destination)` and `(awaiting date)`.

Reply:
Okay, that makes sense, wasn't sure of the flow for a supplier. This makes sense though!